di_website/blog/models.py

```diff
@@ -115,6 +115,12 @@ class BlogArticlePage(TypesetBodyFootnoteMixin, HeroMixin, Page):
             PageChooserPanel('internal_author_page', TeamMemberPage),
             StreamFieldPanel('other_authors')
         ], heading="Author information"),
+        MultiFieldPanel([
+            FieldPanel('call_to_action_title'),
+            FieldPanel('call_to_action_body'),
+            FieldPanel('call_to_action_button_text'),
+            FieldPanel('call_to_action_button_url'),
```

Comment:
Give these simpler headings since they're grouped under one Panel ... you can lose the `call to action` bit ... i.e `FieldPanel('call_to_action_title', heading='Title')`

Reply:
Added simpler headings